### PR TITLE
Create mounted component: footer new id

### PR DESCRIPTION
**Pull request by gisantin**
**CHANGELOG** :memo:

* Creation of footer (mounted component) based on GetNinjas new id layout.

### CODE REVIEW

**Comment by yvessacurae** [comment]
🚀

File: src/yellow/components/_footer.scss [hunk 1]
```diff
@@ -130,20 +131,27 @@
   }
 
   &__institutional {
+    align-items: center;
     border-top: 1px solid $color-interface-shade-dark;
```
Comment (TrevelinT) [inline src/yellow/components/_footer.scss]: Use `$size-border-width-thin`

File: src/yellow/components/_footer.scss [hunk 2]
```diff
@@ -130,20 +131,27 @@
...
-    flex-direction: column;
-    margin-top: $size-spacing-l;
-    padding: $size-spacing-s $size-spacing-xs;
+    flex-direction: row;
+    justify-content: space-between;
+    padding: $size-spacing-m 0;
+
+    &--items {
+      display: block;
+      margin-bottom: 48px;
```
Comment (TrevelinT) [inline src/yellow/components/_footer.scss]: Change to `$size-spacing-l`

File: src/yellow/components/footer/_index.scss [hunk 3]
```diff
@@ -5,20 +5,129 @@
...
+  }
+
+  &__channels {
+    display: flex;
+    padding: 0;
+    text-transform: uppercase;
+
+    .footer__item {
+      list-style: none;
+      margin-right: 16px;
```
Comment (TrevelinT) [inline src/yellow/components/footer/_index.scss]: Change to `$size-spacing-xs`

File: src/yellow/components/list-link/_index.scss [hunk 4]
```diff
@@ -1,62 +1,125 @@
...
     margin-bottom: $size-spacing-xs;
+    text-align: left;
     text-transform: uppercase;
     width: 100%;
   }
 
   &__wrapper {
     display: flex;
     flex-direction: column;
+    margin-bottom: 48px;
```
Comment (TrevelinT) [inline src/yellow/components/list-link/_index.scss]: Change to `$size-spacing-l`

File: src/yellow/components/list-link/_index.scss [hunk 5]
```diff
@@ -1,62 +1,125 @@
...
-  }
 
   &__item {
+    border-radius: $size-border-radius-s;
     cursor: pointer;
+    display: flex;
+    flex-direction: column;
+    flex-shrink: 0;
+    margin-bottom: $size-spacing-xxxs;
     padding: 8px 0;
```
Comment (TrevelinT) [inline src/yellow/components/list-link/_index.scss]: Change to `$size-spacing-xxxs`

File: src/yellow/components/list-link/_index.scss [hunk 6]
```diff
@@ -1,62 +1,125 @@
...
-    }
+    transition: background-color ease-in-out .3s;
 
     &:hover {
       background-color: $color-interface-alert-light;
 
-      .list__link {
+      .lists__link {
         color: $color-interface-content-extra-dark;
         margin-left: 8px;
```
Comment (TrevelinT) [inline src/yellow/components/list-link/_index.scss]: Change to `$size-spacing-xxxs`

File: src/yellow/components/list-link/_index.scss [hunk 7]
```diff
@@ -1,62 +1,125 @@
...
+    display: flex;
+  }
+
+  &__social-media {
+    align-items: flex-start;
+    display: flex;
+    flex-grow: 1;
+
+    a {
+      margin-right: 8px;
```
Comment (TrevelinT) [inline src/yellow/components/list-link/_index.scss]: Change to `$size-spacing-xxxs`

File: src/yellow/gaiden.scss [hunk 8]
```diff
@@ -70,7 +70,7 @@
 @import 'components/page_banner';
 @import 'components/header-banner';
 @import 'components/brand';
-@import 'components/list';
+// @import 'components/list';
```
Comment (pyymenta) [inline src/yellow/gaiden.scss]: ops, may it be removed?

**Pull request merged**